backend/runners/load_representative_points.py

```diff
@@ -72,7 +72,7 @@ def _insert_representative_population_points(json_features):
             unique=False
         )
         return data
-    except Exception as e:
+    except Exception:
         print('Error inserting representative points')
```

Comment:
why is this a print?

Reply:
This happened before logging was a thing.